src/json/converter/csv/CSVReader.java

```diff
@@ -87,8 +87,9 @@ public void readData(){
 
 				String[] data;
 				try {
+					String cleanLine = line.replaceAll("\u0000", "").replaceAll("Sun Nov 30 00:00:00 CET 2", "");
```

Comment:
This is time zone dependent, so will not work on any system and not any time of the year.